internal/distro/rhel90beta/stage_options.go

```diff
@@ -281,8 +277,7 @@ func bootISOMonoStageOptions(kernelVer string, arch string) *osbuild.BootISOMono
 		RootFS: osbuild.RootFS{
 			Size: 9216,
 			Compression: osbuild.FSCompression{
-				Method:  "xz",
-				Options: comprOptions,
+				Method: "lz4",
```

Comment:
RHEL-90 Beta definition should not be changed as it was already released.

Reply:
Oh yes! I'll change that :) Thanks